src/main/java/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {

```

Comment:
The `Storage` class should encapsulate its data more effectively. The `fruitStorage` map should not be public or final, as this allows for external modification which breaks encapsulation. Consider providing methods to interact with the storage instead of exposing it directly.